cypress/e2e/sync.spec.js

```diff
@@ -64,23 +64,23 @@ describe('Sync', () => {
 			.should('include', 'saves the doc state')
 	})
 
-	it('recovers from a lost connection', () => {
-		let count = 0
-		cy.intercept({ method: 'PUT', url: '**/apps/text/session/create' })
-			.as('createSession')
-		cy.intercept({ method: 'POST', url: '**/apps/text/session/*' }, (req) => {
-			if (count < 4) {
+	it('recovers from a short lost connection', () => {
+		let reconnect = false
+		cy.intercept('**/apps/text/session/*', (req) => {
+			if (reconnect) {
+				req.continue()
+				req.alias = 'alive'
+				reconnect = true
```

Comment:
This seems somewhat redundant as reconnect is already checked to be truethy in line 70

Or am I missing something?